definitions/checks/foreman_openscap/report_associations.rb

```diff
@@ -0,0 +1,29 @@
+module Checks
+  module ForemanOpenscap
+    class ReportAssociations < ForemanMaintain::Check
```

Comment:
The check should describe what's wrong about the state, if it fails: what about something like `InvalidReportAssociations`